source/frontend/src/features/projects/list/ProjectFilter/ProjectFilter.tsx

```diff
@@ -0,0 +1,124 @@
+import { ResetButton, SearchButton } from 'components/common/buttons';
+import { Input, Select } from 'components/common/form';
+import { FilterBoxForm } from 'components/common/styles';
+import { PROJECT_STATUS_TYPES, REGION_TYPES } from 'constants/API';
+import { ProjectStatusTypes } from 'constants/projectStatusTypes';
+import { IProjectFilter } from 'features/projects/interfaces';
+import { Formik } from 'formik';
+import useLookupCodeHelpers from 'hooks/useLookupCodeHelpers';
+import React, { useEffect, useState } from 'react';
+import { Col, Row } from 'react-bootstrap';
+import styled from 'styled-components';
+import { mapLookupCode } from 'utils';
+
+export interface IProjectFilterProps {
+  filter?: IProjectFilter;
+  setFilter: (filter: IProjectFilter) => void;
+}
+
+export const defaultFilter: IProjectFilter = {
+  projectName: '',
+  projectNumber: '',
+  projectRegionCode: '',
+  projectStatusCode: ProjectStatusTypes.Active,
+};
+
+/**
+ * Filter bar for Projects.
+ * @param {IProjectFilterProps} props
+ */
+export const ProjectFilter: React.FunctionComponent<
+  React.PropsWithChildren<IProjectFilterProps>
+> = ({ filter, setFilter }) => {
+  const [selectedStatus, setInitialSelectedStatus] = useState<string>(ProjectStatusTypes.Active);
```

Comment:
I don't think using state here makes sense. If you want the default value to be active I would recommend setting that in whatever parent is passing the filter prop here.

Reply:
passed with prop